crates/router/src/connector/payme/transformers.rs

```diff
@@ -312,7 +314,7 @@ impl From<(&SaleQuery, u16)> for types::ErrorResponse {
             reason: sale_query_response.sale_error_text.clone(),
             status_code: http_code,
             attempt_status: None,
-            connector_transaction_id: None,
+            connector_transaction_id: sale_query_response.sale_error_code.clone(),
```

Comment:
Do we get connector_transaction_id from sale_error_code?

Reply:
No. This must be sale_query_response.sale_payme_id.